logdna/response_types.go

```diff
@@ -105,6 +113,21 @@ type indexRateAlertResponse struct {
 	Enabled        bool                          `json:"enabled,omitempty"`
 }
 
+func (indexRateAlert indexRateAlertResponse) ConvertWebhook() ([]interface{}) {
```

Comment:
I'd implement this as a regular function, the name of this function does not match with other functions 
```js
func mapIndexRateAlertWebhookToSchema(indexRateAlert indexRateAlertResponse) ([]interface{}) {
	webhooks := make([]interface{}, 0)

	for _, webhook := range indexRateAlert.Channels.Webhook {
		w := make(map[string]interface{})

		w["bodytemplate"] = webhook.BodyTemplate
		w["headers"]      = webhook.Headers
		w["method"]       = webhook.Method
		w["url"]          = webhook.URL

		webhooks = append(webhooks, w)
	}
	return webhooks
}
```

Reply:
FYI Arani: `ConvertWebhook` is a public method whereas `convertWebhook` is a package private one.

Reply:
I thought we were talking about having this part of the Indexrate alert response. i will make the change